### ISSUE TITLE
live (not DVR): seek to live position before resuming

### ISSUE THREAD

**Issue by foliovision** [foliovision/freedom-video-player#828]
Original issue link: [#1190 live (not DVR): seek to live position before resuming](https://github.com/flowplayer/flowplayer/issues/1190)

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/168338?v=4' /> [phloxic](https://github.com/phloxic) commented on May 10, 2017

In my opinion this makes more sense. And the behavior can become slightly erratic if the player tries to play some cached segments outside the live window after a long pause.
https://flowplayer.org/forum/#!/open_word:pause-play-back-to

There is nothing to be done regarding native playback (duration == Infinity), except when a DVR stream is shown in a live UI, but Flash HLS is affected.

Easy to implement in a beforeresume callback.

@nnarhinen - let me know whether you are ok with this, I can submit a patch then.

Somewhat related:
https://github.com/flowplayer/flowplayer/issues/1189

----

<img width='32' height='32' src='https://avatars.githubusercontent.com/u/168338?v=4' /> [phloxic](https://github.com/phloxic) commented on May 15, 2017

Basically this does it:
```js
api.on("beforeresume", function (e, api) {
    if (api.video.duration && api.video.duration !== Infinity) {
      api.seek(api.video.duration);
   }
});
```
for Flash HLS. Not sure where the appropriate place for this would be in the code. @nnarhinen
